src/static/riot/competitions/detail/submission_modal.tag

```diff
@@ -1,177 +1,125 @@
 <submission-modal>
-    <div class="ui {three: submission.admin}{two: !submission.admin} item small green secondary pointing menu">
-        <div class="active item" data-tab="downloads">Downloads</div>
-        <div class="item" data-tab="logs">Logs</div>
-        <div class="item" data-tab="admin" if="{submission.admin}">Admin</div>
+    <div class="ui large green pointing menu">
+        <div class="active item" data-tab="downloads">DOWNLOADS</div>
+        <div class="item" data-tab="logs">LOGS</div>
+        <div class="item" data-tab="admin" if="{submission.admin}">ADMIN</div>
     </div>
-    <div class="ui tab active" data-tab="downloads">
+    <div class="ui tab active modal-tab" data-tab="downloads">
         <div class="ui relaxed centered grid">
-            <div class="ui fourteen wide column">
-                <div class="ui list">
-                    <div class="item">
-                        <a href="{ submission.data_file }">My Submission</a>
-                    </div>
-                    <div class="item">
-                        <a href="{ submission.result }">Output from prediction step</a>
-                    </div>
-                    <div class="item">
-                        <a href="#">Output from scoring step</a>
-                    </div>
-                    <div class="item">
-                        <a href="#">Private output from scoring step</a>
-                    </div>
-                </div>
+            <div class="ui fifteen wide column">
+                <div class="ui horizontal divider"></div>
+                <table class="ui very basic table" id="downloads">
+                    <thead>
+                    <tr>
```

Comment:
what do you think about indentation with `<thead>`s and such? Should we indent this style, or with an extra indent for `<tr>`s?

Reply:
This styling is what Pycharm defaults to. It does seem a little strange to me. Indenting `<tr>`s makes more sense to me, it just ins't Pycharm's default behavior, so I have left it alone.